doc/ecs/ecs_basics.md

```diff
@@ -27,7 +27,14 @@ Der Zustand einer Entität wird also über ihre Components bestimmt, und ihr Ver
 
 ![Struktur ECS](img/ecs.png)
 
-*Anmerkung: Das abgebildete UML ist nicht aktuell, es wird in #496 aktualisiert.*
+Game ist die Basisklasse von der alles ausgeht. Die Methode `Game#render` ist die Game-Loop. Die Klassen `Entity`, `Component` und `ECS_System` sind die Implementierungen des ECS.
+
+Die LevelAPI generiert, zeichnet und speichert das aktuelle Level. Mehr zum Thema Level erfahren Sie [hier](../level/readme.md).
+
+*Anmerkung:* Das UML ist für bessere Lesbarkeit gekürzt.
+
```

Comment:
Anmerkung Color Coding erklären
- gelb s.u
- weiß = ECS
- grün = Main 
- orange = example implementations
- red = level grams


nur in schön formuliert